pylace/src/transition.rs

```diff
@@ -108,6 +108,53 @@ impl StateTransition {
     fn __repr__(&self) -> String {
         self.to_string()
     }
+
+    /// Create a dictionary of commonly used state transitions sets.
+    #[staticmethod]
+    fn common_transition_sets() -> std::collections::HashMap<String, Vec<Self>>
+    {
+        let mut sets = std::collections::HashMap::new();
+
+        // Add default state transitions
+        sets.insert(
+            String::from("default"),
+            lace::cc::transition::DEFAULT_STATE_TRANSITIONS
+                .iter()
+                .map(|x| Self(x.clone()))
+                .collect(),
+        );
+
+        // Add state transitions to quick movements
+        sets.insert(
+            String::from("sams"),
+            vec![
+                Self(lace::cc::transition::StateTransition::ViewAlphas),
+                Self(lace::cc::transition::StateTransition::RowAssignment(
+                    RowAssignAlg::Sams,
+                )),
+                Self(lace::cc::transition::StateTransition::ViewAlphas),
+                Self(lace::cc::transition::StateTransition::RowAssignment(
+                    RowAssignAlg::Sams,
+                )),
+                Self(lace::cc::transition::StateTransition::ViewAlphas),
+                Self(lace::cc::transition::StateTransition::ComponentParams),
+                Self(lace::cc::transition::StateTransition::RowAssignment(
+                    RowAssignAlg::Sams,
+                )),
+                Self(lace::cc::transition::StateTransition::ComponentParams),
+                Self(lace::cc::transition::StateTransition::RowAssignment(
+                    RowAssignAlg::Gibbs,
```

Comment:
Is this supposed to be a column reassignment? If so, we should only do one Gibbs column reassignment and no slice. There is also no state alpha transition.

Reply:
Whoops, yeah, I copied that incorrectly.